README.md

```diff
@@ -83,16 +62,81 @@ class ExampleGeoMultipleChoiceForm(forms.ModelForm):
 
 - `MAP_ID` sets the html `id` for your widget map. Useful if you have multiple maps on the page.
 - `MAP_HEIGHT` and `MAP_WIDTH` allow you to size the widget in your form. Must be used together.
+- This example uses the default values for `MAP_LAYER_STYLE` and `MAP_LAYER_STYLING`.
```

Comment:
Spitballing off of this sentence, it might be good to add an extra bullet explaining which options are required (or, if all options have reasonable defaults, that no options are required).